apps/admin/backend/src/app.cvr_files.test.ts

```diff
@@ -176,6 +177,10 @@ test('happy path - mock election flow', async () => {
 
   expect(await apiClient.getCastVoteRecordFileMode()).toEqual('unlocked');
 
+  const availableCastVoteRecordFiles3 =
+    await apiClient.listCastVoteRecordFilesOnUsb();
+  expect(availableCastVoteRecordFiles3).toMatchObject([]);
+
```

Comment:
@jonahkagan I needed to add this assertion. Above, there's an `removeUsbDrive` call. Below, there's an `insertUsbDrive` call. Each of those calls set up `expectRepeatedCalls`. But it seems like `expectRepeatedCalls` expects _one or more_ calls, when in this case I think expecting _zero or more_ calls would be more useful. Do you think this would be useful to add to `mockFunction`? It would also cover some of the other cases in these tests where I have to add ` usbDrive.status.expectRepeatedCallsWith().resolves({ status: 'no_drive' });` at the beginning. The `mockUsbDrive` could initialize expecting zero or more calls resolving to `no_drive`.

Reply:
If not sure I understand why there could be 0 or more calls in this case. Is it non-deterministic whether `usbDrive.status` is called?

Reply:
Because `usbDrive.status` is only called when some `app.getUsbDriveStatus` is called or when it's called from the exporter, so sometimes it's not called if we don't explicitly poll `app.getUsbDriveStatus`.

This issue is non-existent in the frontends because polling is set up, and anytime we run a test the polling will surely hit the calls at least once. But we're not polling from the backend, so if we don't check status explicitly, it never gets called.

Reply:
In other words, it's deterministic but not automatic in the backend case.

Reply:
Are we able to know if a backend test is going to call `app.getUsbDriveStatus`? Can we just only add the mock expectation if needed?